### PR TITLE
docs: added how to contribute to OpenSource without coding blog

### PR DESCRIPTION
**Pull request by Serendipity96**
docs: added how to contribute to OpenSource without coding blog

ps:
I'm not a developer, I want to contribute to an OpenSource project. What should I do?
So I wrote a blog about how to contribute to OpenSource without coding. I hope to help others who want to contribute to Apache APISIX.

### CODE REVIEW

**Comment by Serendipity96** [comment]
@membphis  What do you think?

**Review by membphis — APPROVED** [review]
It would be cool if we can add an English version.

File: website/blog/2021-09-09-how-to-contribute-to-an-OpenSource-without-coding.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+description: 本文介绍了很多不写代码为开源项目贡献的方式，可以通过写文章、做视频、对外分享、建立本地社区、积极参与社区讨论等方式参与到开源项目中。
+tags: [community]
+---
+
+>
+> [@Serendipity96](https://github.com/Serendipity96) ，Apache APISIX contributor。
+>
+<!--truncate-->
+
+笔者为开源项目贡献过几个 pr，提交过几个 issue，直到现在还清晰记得几年前第一个 pr 被 merge 时的兴奋，体验到了参与开源的乐趣。但是只停留在了 contributor 阶段，没有下文了，因为笔者后来不做研发了，**认为不熟悉代码、不能修 bug 就没有贡献的机会了。**
```
Comment (tao12345666333) [inline website/blog/2021-09-09-how-to-contribute-to-an-OpenSource-without-coding.md]: `pr` should be capitalized

**Review by tao12345666333 — APPROVED** [review]
LGTM

**Comment by Serendipity96** [comment]
> It would be cool if we can add an English version.

Good idea

File: website/blog/2021-09-09-how-to-contribute-to-an-OpenSource-without-coding.md (hunk 1 above)
Comment (Serendipity96) [reply website/blog/2021-09-09-how-to-contribute-to-an-OpenSource-without-coding.md]: thanks

**Comment by wang-yeliang** [comment]
excellent!

**Review by iamayushdas — APPROVED** [review]

**Pull request merged**